.github/workflows/publish_bundle_acceptance.yml

```diff
@@ -0,0 +1,28 @@
+name: Publish bundle.js to acceptance
+
+on: 
+  workflow_dispatch:
+  branches:
+    - acceptance
+  paths-ignore:
+  - 'CHANGELOG.md'
+  - 'package.json'
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-node@v3
+      with: 
+      node-version: 16.11.1
+      - name: build
+      run: yarn build
+
+    - name: bundle
+    run: yarn bundle
+
+    - name: upload
+    env:
+      AZURE_BLOB_ACCOUNT: ${{ secrets.AZURE_BLOB_ACCOUNT_EDGE }}
```

Comment:
Is it intentional to use the Azure blob account secret for edge while being in the pulbish_budnle_acceptance?